test/commands/multi.spec.js

```diff
@@ -64,9 +64,9 @@ describe("The 'multi' method", function () {
                     // Provoke an error at queue time
                     multi1 = client.MULTI();
                     multi1.mset("multifoo", "10", "multibar", "20", helper.isString("OK"));
-                    multi1.set("foo2", helper.isError());
-                    multi1.incr("multifoo", helper.isNumber(11));
-                    multi1.incr("multibar", helper.isNumber(21));
+                    multi1.set("foo2");
+                    multi1.incr("multifoo");
+                    multi1.incr("multibar");
```

Comment:
yeah the `redis@2.4.5` build that has been ported to Windows has significantly different behavior for `multi/exec` -- I'm not too surprised, since there's a bunch of conditional logic surrounding this.